docs/_generate/index.html

```diff
@@ -105,10 +105,11 @@
       <p>The intended audience of this specification includes:
       <ul>
       <li>Advanced designers and cartographers who want to write styles by hand rather than use <a href='https://www.mapbox.com/studio'>Mapbox Studio</a></li>
-      <li>Developers using style-related features of <a href='https://www.mapbox.com/mapbox-gl-js/'>Mapbox GL JS</a> or the Mapbox <a href='https://www.mapbox.com/ios-sdk/'>iOS</a> or <a href='https://www.mapbox.com/android-sdk/'>Android</a> SDKs</li>
+      <li>Developers using style-related features of <a href='https://www.mapbox.com/mapbox-gl-js/'>Mapbox GL JS</a> or the <a href='https://www.mapbox.com/android-sdk/'>Mapbox Android SDK</a></li>
       <li>Authors of software that generates or processes Mapbox styles.</li>
       </ul>
       </p>
+      <p>Developers using the <a href='https://www.mapbox.com/ios-sdk/'>Mapbox iOS SDK</a> or <a href='https://github.com/mapbox/mapbox-gl-native/tree/master/platform/macos/'>Mapbox macOS SDK</a> should consult the <a href="https://www.mapbox.com/ios-sdk/api/3.4.0-beta.6/">iOS SDK API reference</a> for platform-appropriate documentation of style-related features.</p>
```

Comment:
Can we use a link for the iOS SDK API that redirects to the latest documentation, rather than one which will need to be updated whenever we release a new version?